src/main-menu.ts

```diff
@@ -0,0 +1,44 @@
+import { LitElement, css, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { game } from './game';
+
+@customElement('main-menu')
+export class MainMenu extends LitElement {
+  static styles = css`
+    :host {
+      width: 100vw;
+      height: 100vh;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      flex-direction: column;
+    }
+  `;
+
+  @property({ type: Boolean })
+  settingsOpen = false;
+
+  startGame() {
```

Comment:
nit: private (here and below)

also can we put the order as
- render
- renderSettings
- startGame